src/ZODB/DB.py

```diff
@@ -385,23 +366,53 @@ def __init__(self, storage,
+             :class:`~ZODB.FileStorage.FileStorage.FileStorage`
+             storage or ``None`` to use a constructed
+             :class:`~ZODB.MappingStorage.MappingStorage`.
+        :param int pool_size: expected maximum number of open connections.
+             Warnings are logged when this is exceeded and critical
+             messages are logged if twice the pool size is exceeded.
+        :param seconds pool_timeout: Maximum age of inactive connections
+             When a connection has remained unused in a connection
+             pool for more than pool_timeout seconds, it will be
+             discarded and it's resources released.
+        :param objects cache_size`: target maximum number of non-ghost
+             objects in each connection object cache.
+        :param int cache_size_bytes`: target total memory usage of non-ghost
+             objects in each connection object cache.
+        :param int historical_pool_size`: expected maximum number of total
             historical connections
-          - `historical_cache_size`: target size of Connection object cache for
-            historical (`at` or `before`) connections
-          - `historical_cache_size_bytes` -- similar to `cache_size_bytes` for
-            the historical connection.
-          - `historical_timeout`: minimum number of seconds that
-            an unused historical connection will be kept, or None.
-          - `xrefs` - Boolian flag indicating whether implicit cross-database
-            references are allowed
+        :param objects historical_cache_size`: target maximum number
+             of non-ghost objects in each historical connection object
+             cache.
+        :param int historical_cache_size_bytes`: target total memory
+             usage of non-ghost objects in each historical connection
+             object cache.
+        :param seconds historical_timeout: Maximum age of inactive
+             historical connections.  When a connection has remained
+             unused in a historical connection pool for more than pool_timeout
+             seconds, it will be discarded and it's resources
+             released.
+        :param str database_name: The name of this database in a
+             multi-database configuration.  The name is used when
+             constructing cross-database references ans when accessing
+             database connections fron other databases.
+        :param dict databases: dictionary of database name to
+             databases in a multi-database configuration. The new
+             database will add itself to this dictionary. The
+             dictionary is used when getting connections in other databases.
+        :param boolean xrefs: Flag indicating whether cross-database
+            references are allowed from this database to other
+            databases in a multi-database configuration.
+        :param int large_record_size: When object records are saved
```

Comment:
It's *very* theme dependent. Some of them look better than others. Most of them don't seem to do anything. If it looks bad in your theme then so be it. Thanks for trying.